extension/src/popup/helpers/fetch.ts

```diff
@@ -4,7 +4,7 @@ export const fetchJson = async <T>(url: string, options?: RequestInit) => {
     throw new Error(res.statusText);
   }
 
-  if (res.headers.get("content-type") !== "application/json") {
+  if (!res.headers.get("content-type")?.includes("application/json")) {
```

Comment:
Seems like a fastify version tweaked this header slightly(added the charset) so maybe it's better to try to be looser with the content type here?

Reply:
Oh interesting - looks like we only use this in `useScanTx` (so far), so should be safe to change this